server/generated/openapi/service.swagger.json

```diff
@@ -5066,7 +5214,7 @@
           "items": {
             "$ref": "#/definitions/v1CalendarSpec"
           },
-          "description": "Any timestamps matching any of exclude_* will be skipped."
+          "description": "Any timestamps matching any of exclude_* will be skipped.\n\nuse exclude_structured_calendar"
```

Comment:
Was this an intentional update? Only asking as it seems unrelated 😅

Reply:
Hah yeah I'm not sure what this is exactly. These files are generated so I just kept what was there.